rubicon_ml/client/mixin.py

```diff
@@ -382,16 +396,23 @@ def dataframe(self, name=None, id=None):
                 )
 
             dataframe = dataframes[-1]
+            return dataframe
```

Comment:
same comment as with the experiment and artifact getting function - this first part needs to be inside the try too to handle getting by name and ID